### PR TITLE
RELATED: RAIL-286 Added metadata visualisation API

### PR DESCRIPTION
**Pull request by zbycz**
and fixed expect to be called in ok result

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-js-opensource-repository-check/37/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-js-unit-tests-zuul-el7/274/ : SUCCESS in 54s

File: src/metadata.js [hunk 1]
```diff
@@ -131,6 +131,17 @@ export function getObjectUsingMany(projectId, uris, options = {}) {
 }
 
 /**
+ * Reutrns all visualisations metadata in a project specified by projectId param
```
Comment (vaclavbohac) [inline src/metadata.js]: Returns

File: src/metadata.js [hunk 2]
```diff
@@ -131,6 +131,17 @@ export function getObjectUsingMany(projectId, uris, options = {}) {
...
+ *
+ * @method getVisualisations
+ * @param projectId Project identifier
```
Comment (vaclavbohac) [inline src/metadata.js]: `* @param {string} projectId project identifier`

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-js-opensource-repository-check/38/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-js-unit-tests-zuul-el7/275/ : SUCCESS in 53s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-js-opensource-repository-check/39/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-js-unit-tests-zuul-el7/276/ : SUCCESS in 51s

**Comment by zbycz** [comment]
Thanks - both comments fixed also other occurences of the same typo.
Plus added typescript spec.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-js-opensource-repository-check/40/ : SUCCESS in 6s
- https://ci.intgdc.com/job/gooddata-js-unit-tests-zuul-el7/277/ : FAILURE in 58s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-js-opensource-repository-check/41/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-js-unit-tests-zuul-el7/278/ : SUCCESS in 51s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-js-opensource-repository-check/42/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-js-unit-tests-zuul-el7/279/ : SUCCESS in 2m 08s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-js-merge-compliance-check/42/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-js-unit-tests-zuul-el7/280/ : SUCCESS in 4m 54s

**Pull request merged**